lib/share.c

```diff
@@ -39,6 +39,7 @@ curl_share_init(void)
   if(share) {
     share->magic = CURL_GOOD_SHARE;
     share->specifier |= (1<<CURL_LOCK_DATA_SHARE);
+    share->max_ssl_sessions = 8;
```

Comment:
It's not the fault of this patch, but `8` should be defined with a name or at the very least be given a comment explaining the magic number.

Reply:
I have added a comment that this is the default cache size.